recipes-bsp/uefi/tegra-uefi-capsules_36.3.0.bb

```diff
@@ -26,9 +26,10 @@ do_compile() {
     # BUP generator really wants to use 'boot.img' for the LNX
     # partition contents
     tegraflash_populate_package ${IMAGE_TEGRAFLASH_KERNEL} boot.img ${@tegra_bootcontrol_overlay_list(d, bup=True)}
-    cp "${IMAGE_TEGRAFLASH_ESPIMG}" ./esp.img
+    # Generate dummy ESP image, as it's not included into the final BUP image
+    dd if=/dev/zero of=./esp-dummy.img bs=1 count=65536
```

Comment:
Rather than creating a dummy image, I think it would make more sense to simply remove the reference to the `esp.img` file from the flash layout XML altogether. That should eliminate the need for changing the signature of the function that creates the flash config.

Reply:
@madisongh thanks for suggestion, pushed updated version